Source/Core/AudioCommon/Mixer.h

```diff
@@ -47,41 +46,125 @@ class CMixer
 	float GetCurrentSpeed() const { return m_speed.load(); }
 	void UpdateSpeed(float val) { m_speed.store(val); }
 
+	static inline float lerp(float sample1, float sample2, float fraction) {
```

Comment:
I feel like this is a common enough function that it is probably used elsewhere, and we could extract it to a common header. Maybe also the following s16<->float (although this takes into account the -32768 value instead of truncating it to -32767 or stretching the positive side to 32768).